rokka/client.go

```diff
@@ -66,9 +83,32 @@ func (c *Client) Call(req *http.Request, v interface{}) error {
 
 	decoder := json.NewDecoder(resp.Body)
 
-	return decoder.Decode(&v)
+	if resp.StatusCode < 400 {
```

Comment:
That's a matter of preference, really: I use instead of `400` for http status codes the constants defined in package [`net/http`](https://godoc.org/net/http#pkg-constants): `http.StatusBadRequest`